[serverless] Updates Data Quality Dashboard UI (#5718)
elastic/security-docs: docs/serverless/dashboards/data-quality-dash.mdx

To customize which indices are checked when you click **Check all**, [change the current data view](((security-guide))/data-views-in-sec.html).
</DocCallOut>

> * **Check a single index**: To check a single index, click the **Check index** button under **Actions**. Checking a single index is faster than checking all indices.

Reviewer: ```suggestion
* **Check a single index**: To check a single index, click the **Check index** button under **Actions** column. Checking a single index is faster than checking all indices.
```

Click a node in the treemap to expand the corresponding index.

## Learn more about checked index fields

> After an index is checked, `Pass` or `Fail` appears in its **Result** column. `Fail` indicates mapping problems in an index. To view index details, including which fields weren't successfully mapped, click the **Index details** button under **Actions**.

Reviewer: ```suggestion
After an index is checked, `Pass` or `Fail` appears in its **Result** column. `Fail` indicates mapping problems in an index. To view index check details, including which fields weren't successfully mapped, click the **View check details** button under **Actions**.
```

![An expanded index with some failed results in the Data Quality dashboard](../images/data-quality-dash/-dashboards-data-qual-dash-detail.png)

> The index details flyout provides more information about the status of fields in that index. Each of its tabs describe fields grouped by mapping status.

Reviewer: ```suggestion
The index check flyout provides more information about the status of fields in that index. Each of its tabs describe fields grouped by mapping status.
```

**Export results for one index**:

> 1. View details for an index that has at least one incompatible field by clicking the **Index details** button under **Actions**.

Reviewer: ```suggestion
1. View check details for an index that has at least one incompatible field by clicking the **View check details** button under **Actions**.
```

> 1. View check details for an index that has at least one incompatible field by clicking the **View check details** button under **Actions**.

Reviewer 2: The first part of this sentence sounds a bit awkward. It might be because "check details" is being used as a noun. Would something like this work:

```suggestion
1. View details for a checked index that has at least one incompatible field by clicking the **View check details** button under **Actions**.
```
